From c289098b259d4063fb3bf809a48c69ab9aa8bc25 Mon Sep 17 00:00:00 2001
From: Julian Andres Klode <julian.klode@canonical.com>
Date: Thu, 11 Jan 2018 09:11:05 +0100
Subject: [PATCH 2/6] dput.cf: Adjust upload paths for Ubuntu and ppas

The default upload paths changed a while ago, let's bring that
up to date.

LP: #1340130
---
 dput.cf | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/dput.cf b/dput.cf
index 764025b..d328686 100644
--- a/dput.cf
+++ b/dput.cf
@@ -78,14 +78,14 @@ pre_upload_command	= /usr/share/dput/helper/security-warning
 [ubuntu]
 fqdn			= upload.ubuntu.com
 method			= ftp
-incoming		= /
+incoming		= /ubuntu
 login			= anonymous
 
 [ppa]
 fqdn			= ppa.launchpad.net
 method			= ftp
 # replace <launchpad-id> with your Launchpad ID
-incoming		= ~<launchpad-id>/ubuntu
+incoming		= ~<launchpad-id>
 login			= anonymous
 
 [mentors]
-- 
2.15.1

